src/sections/@dashboard/sidebar/MenuContent.js

```diff
@@ -1,87 +1,37 @@
 /* eslint-disable react/no-array-index-key */
 import { ExpandLess, ExpandMore } from '@mui/icons-material';
-import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet';
-import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
-import QrCodeScannerIcon from '@mui/icons-material/QrCodeScanner';
-import SupportAgentIcon from '@mui/icons-material/SupportAgent';
 import {
   Collapse,
   List,
   ListItem,
   ListItemButton,
   ListItemIcon,
   ListItemText,
-  MenuItem,
   Stack,
   styled,
 } from '@mui/material';
 import Image from 'next/image';
 import { useEffect, useState } from 'react';
 import MenuPopover from '~/components/MenuPopover';
-
-const MAIN_LIST_ITEMS = [
-  {
-    id: 'general',
-    text: 'general',
-    icon: <HomeRoundedIcon />,
-    subItems: [],
-  },
-  {
-    id: 'transaction',
-    text: 'transaction',
-    icon: <AccountBalanceWalletIcon />,
-    subItems: [
-      { id: 'payment', text: 'payment' },
-      { id: 'pending', text: 'pending' },
-    ],
-  },
-  {
-    id: 'extension',
-    text: 'extension',
-    icon: <QrCodeScannerIcon />,
-    subItems: [],
-  },
-  {
-    id: 'integrate',
-    text: 'integrate',
-    icon: (
-      <Image
-        src="/images/interface-link--create-hyperlink-link-make-unlink.svg"
-        width={20}
-        height={20}
-        alt="link_icon"
-        style={{ color: '#666A72' }}
-      />
-    ),
-    subItems: [],
-  },
-  {
-    id: 'contact',
-    text: 'contact',
-    icon: <SupportAgentIcon />,
-    subItems: [],
-  },
-];
+import { useAppSelector } from '~/redux/hook';
 
 export default function MenuContent({ drawerOpen, ...props }) {
```

Comment:
drawerOpen is boolean, name it `isDrawerOpen`

Reply:
Dạ